test/builtin_dataset_mocks.py

```diff
@@ -965,8 +965,8 @@ def food101(info, root, config):
     return num_samples_map[config.split]
 
 
-# @register_mock
-def dtd(info, root, config):
+@register_mock(configs=combinations_grid(split=("train", "val", "test"), fold=(1, 4)))
```

Comment:
I agree we probably don't need to test all folds, but maybe we can do the first, the last and one from the middle?

```suggestion
@register_mock(configs=combinations_grid(split=("train", "val", "test"), fold=(1, 4, 10)))
```